src/stages/fillSlot.ts

```diff
@@ -445,6 +483,7 @@ async function runRetryCheck<T, G>(
   dispatch: Dispatch,
   getState: () => WolfState,
   convoStorageLayer: G,
+  slots: Slot<G>[],
   abilities: Ability<T, G>[],
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1